xcell/cls/data.py

```diff
@@ -24,6 +24,9 @@ def __init__(self, data_path='', data={}, override=False):
 
         os.makedirs(self.data['output'], exist_ok=True)
         self._check_yml_in_outdir(override)
+        self.tr_matrix = None
+        self.cl_tracers = {True: None, False: None}
```

Comment:
I was wondering what the `True` and `False` were. Now I see they are the bools for the `wsp` option. I think we should change this to something more readable, but since it will require modifying more parts of the code, we can leave it for a later PR

Reply:
Changed to `wsp` and `no_wsp`